### DISCUSSION TITLE
Why "styled: false"?

### DISCUSSION THREAD

**Discussion by acalvino4** [saadeghi/daisyui#1518] [Q&A] [answered]
So the config has `styled` option - but if I set that to "false", what's the point of using DaisyUI at all? docs say

> If it's false, components will have no color and no visual style

but what else does daisyui provide _besides_ color and style? Do some styles remain? which ones?

**Comment by saadeghi** [comment, marked as answer]
This is basically for people who don't want daisyUI design decisions (themes, color names, border radius, etc.) but they want the skeleton (layout, sizing, positioning) of components.  

For example, let's say you don't want the daisyUI color names (primary, secondary, etc.) And you want to use Tailwind CSS color name utility classes. You can set `styled` to `false` and use something like `<button class="btn bg-indigo-600 text-white">`

If you want to know the exact CSS of styled/unstyled components, here are the exact CSS files for each one:
- https://github.com/saadeghi/daisyui/tree/master/src/components
- https://github.com/saadeghi/daisyui/tree/master/src/utilities

**Comment by acalvino4** [comment]
This seems like a somewhat blurry line to draw, as in a sense all css is often considered to be "styles", and I don't think I feel perfectly clear on how the distinction would apply across the project. And recently I've been feeling somewhat like I want the opposite (keeping daisy's semantic theming and css vars, but ditching the layout/styles/etc), though I haven't opened a feature request because I'm not really completely sure how to describe the line either....

I like the ability to use daisy for theming, as well as for getting going quickly, but have also quickly found the need to create more overrides than seems right. But still somewhat working out how I feel. I appreciate the ability to get going quickly with pre-built components, but it also seems somewhat of an anti-pattern for tailwind.

**Reply by saadeghi** [reply]
It is indeed a blurry line. Sometimes it's not really easy to decide which line of CSS should be considered as a design decision.  
Based on my knowledge, almost everyone is using daisyUI as-is (with all components `styled` and with default built-in themes) however daisyUI's component designs are opinionated (like all other UI libraries) so I think the it should be possible to opt-out of design decisions somehow.

Here you can see how components will look like with [`styled=false`](https://play.tailwindcss.com/Fu5k9I4U6n) comparing to [`styled=true`](https://play.tailwindcss.com/8qSyMCeyGk?file=config)

About the opposite scenario you mentioned:  
If you want to have daisyUI colors and themes without the components, you can simply install daisyUI, then just use the daisyUI color names. Don't use component classes (`btn`, etc.). Just use `bg-primary` or any other daisyUI colors using utility classes. All unused component styles will be excluded anyway.

**Reply by IAPBenediktZwoelfer** [reply]
would like to take care of all the colors on my own but still use the animations and layout styles from DaisyUI, how cam i accomplish this? I used  styled=false but this disables everything. Thanks!

**Reply by saadeghi** [reply]
@IAPBenediktZwoelfer One way is disabling all themes like this  
https://play.tailwindcss.com/YlIBNojvqX?file=config
```js
module.exports = {
  plugins: [require('daisyui')],
  daisyui: {
    themes: [],
  },
}
```
It removes all the built-in themes.
